index.js

```diff
@@ -42,9 +45,16 @@ const portCheckSequence = function * (ports) {
 
 module.exports = async options => {
 	let ports;
+	let exclusions = [];
```

Comment:
For performance, it should use a `Set` internally since it's faster for checking.